### DISCUSSION TITLE
Choose installation location

### DISCUSSION THREAD

**Discussion by Gabrielquotes2** [minbrowser/min#2489] [Q&A]
I'm trying to install Min but it just 'installs' without asking me where I want it, and I can't find it under C:\Program Files.
This means I don't know how to make a desktop shortcut for it because I can't find the program!
Is there any way to choose where it installs?

**Comment by sriijasdc** [comment]
The only thing i know is the app gets saved under this folderC:\Users\usern\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\PalmerAL

**Comment by mouseclicker** [comment]
Hah, it did not install there for me. I was pretty excited to try this guy out but got to uninstall cause I really want to make sure I know where my things are.
